### PR TITLE
[PTX-22951] Move doPDBValidation and OpVer24_1_0 to common.go

### PR DESCRIPTION
**Pull request by kphalgun-px**
**What this PR does / why we need it**:
This PR moves the doPDBValidation function and the OpVer24_1_0 variable to common.go to enable their use in the TriggerUpdateCluster in the Longevity test suite ([PTX-22914](https://portworx.atlassian.net/browse/PTX-22914)). This PR also renames OpVer24_1_0 to PDBValidationMinOpVersion for brevity. 

**Which issue(s) this PR fixes** (optional)
Closes #PTX-22951

**Special notes for your reviewer**:
I think this PR has minimal changes and is just refactoring, so I did not test it, just reviewed it.

[PTX-22914]: https://portworx.atlassian.net/browse/PTX-22914?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by nikolaypopov — APPROVED** [review]
Thanks for moving this, its a better place for it anyways!

File: tests/common.go [hunk 1]
```diff
@@ -10948,3 +10950,41 @@ func SwitchBothKubeConfigANDContext(cluster string) error {
...
+			if allowedDisruptions > 1 && !isClusterParallelyUpgraded {
+				err := fmt.Errorf("cluster is not parallely upgraded")
+				*mError = multierr.Append(*mError, err)
+				log.Warnf("Cluster not parallely upgraded as expected")
+			}
+			log.Infof("Exiting PDB validation routine")
+			return
+		default:
+			errorChan := make(chan error, 50)
+			ValidatePDB(pdbValue, allowedDisruptions, totalNodes, &isClusterParallelyUpgraded, &errorChan)
```
Comment (svijaykumar-px) [inline tests/common.go:10979]: nit : Since ValidatePDB function is in the same file, you can change it to validatePDB

**Review by svijaykumar-px — APPROVED** [review]

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

File: tests/common.go (hunk 1 above)
Comment (svijaykumar-px) [reply tests/common.go:10979]: Oh I assumed all cases will be written in this file itself. Otherwise we can keep it the same

**Pull request merged**